src/cli/onehost

```diff
@@ -116,26 +110,21 @@ cmd=CommandParser::CmdParser.new(ARGV) do
         Creates a new Host
     EOT
 
-    command :create, create_desc, :hostname, :options=>CREAT_OPTIONS do
+    command :create, create_desc, :hostname, [:path, nil], :options=>CREAT_OPTIONS do
```

Comment:
Use `:file` instead of `:path`. It's already defined as an argument type:

https://github.com/OpenNebula/one/blob/master/src/cli/command_parser.rb#L805-L807